src/main/java/net/earthcomputer/clientcommands/command/GammaCommand.java

```diff
@@ -0,0 +1,36 @@
+package net.earthcomputer.clientcommands.command;
+
+import static com.mojang.brigadier.arguments.DoubleArgumentType.getDouble;
+import static com.mojang.brigadier.arguments.DoubleArgumentType.doubleArg;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.addClientSideCommand;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.*;
+import static net.minecraft.server.command.CommandManager.argument;
+import static net.minecraft.server.command.CommandManager.literal;
```

Comment:
These imports need cleaning up. Static imports should always use `*` wildcards, and go after normal imports.